cves/kernel/CVE-2016-9685.yml

```diff
@@ -147,10 +150,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This bug was discovered by a member of the Red Hat Security team on 
+    2016-11-21. 
+  automated: false
+  contest: false
+  developer: true
```

Comment:
Because the prompt specifies a google developer I would try to answer that question somewhere in this answer (I know you do call out red hat though)